src/OpenLoco/src/Ui/Windows/Vehicle.cpp

```diff
@@ -1966,8 +1966,8 @@ namespace OpenLoco::Ui::Windows::Vehicle
                         {
                             cargoTextHeight += 5;
                         }
-                        drawCargoText(drawingCtx, width, cargoTextHeight, strFormat, body->primaryCargo.qty, body->primaryCargo.type, body->primaryCargo.townFrom);
-                        drawCargoText(drawingCtx, width, cargoTextHeight, strFormat, front->secondaryCargo.qty, front->secondaryCargo.type, front->secondaryCargo.townFrom);
+                        drawCargoText(drawingCtx, width - 24, cargoTextHeight, strFormat, body->primaryCargo.qty, body->primaryCargo.type, body->primaryCargo.townFrom);
```

Comment:
Is it really 24 fixed and not some other variable value.

Reply:
Yeah it's literally offsetting the sprite by -24 a few lines up. Maybe could use a constant? But what to name it?

Reply:
Doh, it's actually much simpler. The text should just be at x=24 at all times.